### ISSUE TITLE
training details for your gla-340M-15B models

### ISSUE THREAD

**Issue by zmj1203** [sustcsonglin/flash-linear-attention#94]
Hi~

Thank your for you contribution.
I have read your paper (GLA and GSA), GitHub page (https://github.com/sustcsonglin/flash-linear-attention) and huggingface page (https://huggingface.co/datasets/fla-hub/). It is a very interesting job!

You said in your paper (GLA) and model card (https://huggingface.co/fla-hub/gla-340M-15B), your gla-340M-15B model is trained  with AdamW (Loshchilov & Hutter, 2018) using a maximum learning rate of 3e-4. The 340M models are trained on 15B tokens with a batch size of 0.5M tokens. 

I have 3 questions:
1. The original SlimPajama dataset (https://huggingface.co/datasets/cerebras/SlimPajama-627B) contains 627B tokens, and you use a 15B subset as training dataset. So where is the 15B training subset dataset? Have you released it? Is it in https://huggingface.co/datasets/fla-hub/slimpajama-test (If not what is the use of this dataset) ?
2. How long you trianed for the gla-340M-15B? By A100 or other cards? 1 A100 or more?
3. If I want to train a smaller models, for example around 130M like mamba or even smaller, which setup in your code I need to modify?

Looking forward to your feedback. Thanks!

**Comment by sustcsonglin** [comment]
1. We didn't release it. 
2. 4 H100 12h should be enough
3. Modify num layer and hidden dimension at https://github.com/sustcsonglin/flash-linear-attention/blob/main/fla/models/mamba/configuration_mamba.py

**Comment by zmj1203** [comment]
OK, I see, thank you very much for your reply!~  :)
